eng/scripts/verify-npm-tags.ps1

```diff
@@ -0,0 +1,49 @@
+param (
+  [Parameter(mandatory = $true)]
+  $pathToArtifacts,
+  [Parameter(mandatory = $true)]
+  $originalDistTags,
+  [Parameter(mandatory = $true)]
+  $intendedTag,
+  [Parameter(mandatory = $true)]
+  $intendedTagVersion
+)
+
+# TODO: delete testing comments below
+#'@azure/eventhubs-checkpointstore-table' alpha, no beta, no GA
+#'@azure/openai' alpha, beta, no GA
+#'@azure/video-indexer-widgets' no alpha, no beta, GA
+#'@azure/template' alpha, beta, GA
+#$pkgProps.PackageId
+
+$ErrorActionPreference = 'Stop'
+$PSNativeCommandUseErrorActionPreference = $true
+
+$pkg = Get-Content -Raw "temp_decompress\package\package.json" | ConvertFrom-Json
```

Comment:
We should pass temp_decompress as a parameter or the fully path instead of assuming it is there.